drafts/mpls-te-topology/draft-busizheng-teas-yang-te-mpls-topology.md

```diff
@@ -17,23 +17,14 @@ author:
     name: Italo Busi
     org: Huawei Technologies
     email: italo.busi@huawei.com
-  -
-    name: Haomian Zheng
-    org: Huawei Technologies
-    email: zhenghaomian@huawei.com
   -
     name: Aihua Guo
     org: Futurewei Inc.
     email: aihuaguo.ietf@gmail.com
   -
     name: Xufeng Liu
-    org: Volta Networks
+    org: IBM Corporation
     email: xufeng.liu.ietf@gmail.com
-  -
-    name: Vishnu Pavan Beeram
-    ins: V. Beeram
-    org: Juniper Networks
-    email: vbeeram@juniper.net
   -
     name: Tarek Saad
     org: Juniper Networks
```

Comment:
please update email to tsaad.net@gmail.com

Reply:
See https://github.com/tsaad-dev/te/pull/201